### PR TITLE
Update wazo-service to use PostgreSQL 13 instead of 11

### PR DESCRIPTION
**Pull request by sopelj**
There are other fixes (I couldn't help myself), but they are isolated in separate commits, so we can cherry-pick or exclude if desired.

### CODE REVIEW

**Comment by sopelj** [comment]
recheck

File: bin/wazo-service [hunk 1]
```diff
@@ -15,44 +15,44 @@ wazo_disabled_file="/var/lib/wazo/disabled"
 
 usage() {
     cat <<-EOF
-	usage : $0 action [services_filter]
-	availables actions :
+	usage : $0 action [service_filters]
```
Comment (fblackburn1) [inline bin/wazo-service:18]: Since it's "one" filter that impact "many" services, is it not more intuitive to add `s` to services instead of filter?

**Review by fblackburn1 — COMMENTED** [review]
Yes please extract them in another branch from master.
Since you like cleanup, can you fix the `wazo-service status wazo` by renaming all `xivo` group in the `main.py` :)

File: bin/wazo-service (hunk 1 above)
Comment (sopelj) [reply bin/wazo-service:18]: English is weird... it's a "service filter" that will "filter services". It's like `service_list` is a list of services, but you wouldn't write "services_list".

Comment (sopelj) [reply bin/wazo-service:18]: Moved to: https://github.com/wazo-platform/wazo-service/pull/26

Comment (sopelj) [reply bin/wazo-service]: And I removed them here as I had to make changes to the commits anyways.

**Comment by sopelj** [comment]
recheck

**Comment by sopelj** [comment]
recheck

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**